pyiron_contrib/executors/executors.py

```diff
@@ -0,0 +1,25 @@
+from concurrent.futures import ProcessPoolExecutor
+
+import cloudpickle
+
+
+def _apply_cloudpickle(fn, /, *args, **kwargs):
+    fn = cloudpickle.loads(fn)
+    return fn(*args, **kwargs)
```

Comment:
I guess you also want to `cloudpickle` the output on the way back.

Reply:
I'm not sure what you mean. Can you suggest a modification to the test that makes it fail without this?

Reply:
I added a suggestion below. The other part I am surprised about is that you do not pickle the arguments, but only the function. Should not we pickle everything together?

Reply:
I guess this is only necessary when the args or return values are themselves things that cannot be normally pickled (eg instances of classes created by a decorator). Tomorrow I'll come back and make the test case much crueler. Hopefully then we'll see the tests fail without your suggestion and similar treatment for the args